recipes/atsas-pipelines/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "atsas-pipelines" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "01610555b39a034cd718812aa70d54322311d568ba4a52ca52e172d3836774e9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - dask-jobqueue
+    - distributed
+    - matplotlib
```

Comment:
Do you need a PyQt GUI? If not, better to depend on `matplotlib-base`.

Reply:
No, no need for the Qt part for this project. Thanks for the pointer, @dopplershift!